File: languages/go/fibonacci.go
package main

import (
	"os"
	"strconv"
)

func fib(n int) int { 
  if n < 2 { return n } 
  return fib(n - 1) + fib(n - 2) 
}

func main(){
	input_number, err := strconv.Atoi(os.Args[1])
	if err == nil {
		println(fib(input_number - 1))
	}
}

File: languages/objectivec/fibonacci.m
#import <stdio.h>

int fib(int n){
	return n < 2 ? n : fib(n-1) + fib(n-2);
}

main(int argc, char *argv[]){
	printf("%d\n", fib(atoi(argv[1]) - 1));
}
